src/integration-tests/integration-test.py

```diff
@@ -504,6 +504,9 @@ def check_jobs_impl() -> Tuple[bool, str, bool]:
 
             for job_id in to_remove:
                 if stop_on_complete_check:
+                    # send the stop marker here to ignore errors after the jobs stops
```

Comment:
I highly recommend against this change.

There are two issues here.
1. This means errors in job shutdown won't be seen.  In the past, this functionality has caught a number of issues including VM reimaging issues and job dependency issues.
2. This prevents checking logs for issues with live repros, which happen *after* jobs are shut down.